hooks/common/theme/theme-build.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 
 NPM_CMD="npm"
-# Run scripts via lando if available
-if [ -e .lando.yml ] && command -v lando &> /dev/null; then
+if [ -n "$DDEV_PROJECT" ] || [ -n "$LANDO_PROJECT" ]; then
+  # If inside a container, use npm directly.
+  NPM_CMD="npm"
+elif [ -e .ddev/config.yaml ] && command -v ddev &> /dev/null; then
+  # If outside a DDEV container, use ddev npm.
+  NPM_CMD="ddev npm"
+elif [ -e .lando.yml ]  && command -v lando &> /dev/null; then
+  # If outside a Lando container, use lando npm.
```

Comment:
Is the "inside/outside" terminology inversed in the comments here? Or am I reading it backwards.

Also, I wonder if we should create a single wrapper command we could run these with, and then wrapper would determine whether to run the command prepended with "ddev" or "lando".